net-libs/libtorrent/libtorrent-0.13.7.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools toolchain-funcs
+
+DESCRIPTION="BitTorrent library written in C++ for *nix"
+HOMEPAGE="https://rakshasa.github.io/rtorrent/"
+SRC_URI="http://rtorrent.net/downloads/${P}.tar.gz"
+
+LICENSE="GPL-2"
+
+# The README says that the library ABI is not yet stable and dependencies on
+# the library should be an explicit, syncronized version until the library
+# has had more time to mature. Until it matures we should not include a soname
+# subslot.
+SLOT="0"
+
+KEYWORDS="~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd ~amd64-linux ~arm-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x64-solaris"
+IUSE="debug libressl ssl test"
```

Comment:
I see you removed ipv6 flag, what is that about?

Reply:
The IPv6 use flag no longer had any effect. The ability to toggle IPv6 support at compile time was removed from the project. It's resulted in a bug referenced further down in the source: https://github.com/rakshasa/rtorrent/issues/732

Reply:
this like that are good to have in the commit message, in case someone asks questions afterwards.

Reply:
Pushed an amended commit message with further information

Reply:
thanks!